specs/phase1/beacon-chain.md

```diff
@@ -530,33 +533,75 @@ def get_indexed_attestation(beacon_state: BeaconState, attestation: Attestation)
     )
 ```
 
+#### `get_committee_count_delta`
+
+```python
+def get_committee_count_delta(state: BeaconState, start_slot: Slot, stop_slot: Slot) -> uint64:
+    """
+    Return the sum of committee counts between ``[start_slot, stop_slot)``.
```

Comment:
`between` may not be accurate here. Since `[` means inclusive

Reply:
How about "between ``[start_slot, stop_slot)``" -> "in range ``[start_slot, stop_slot)``"?

Reply:
looks good